src/pages/Login.vue

```diff
@@ -412,22 +417,24 @@ export default {
      */
     startResendOTPTimer(seconds = 60) {
       this.resendOTPTimer = seconds;
-      const interval = setInterval(() => {
+      this.interval = setInterval(() => {
         this.resendOTPTimer--;
-        if (!this.resendOTPTimer || !this.isPhoneValid()) clearInterval(interval);
+        if (!this.resendOTPTimer) clearInterval(this.interval);
       }, 1000);
     },
     requestOtp() {
       // requests OTP for the first time
       UserAPIService.requestOtp(this.formattedPhoneInput);
       this.requestedOtp = true;
       this.startResendOTPTimer();
+      this.otpInput= "";

```

Comment:
same space issue! Please try looking up for some tool that can automatically correct this or please be mindful while writing the code itself that you do not make this error. doesn't feel like a good use of our time to keep pointing this out :/

Reply:
Yes, I don't know why it is not working will find something else.